decosjoin/api/decosjoin/decosjoin_connection.py

```diff
@@ -150,31 +151,65 @@ def _transform(self, zaken, user_identifier):
                     continue
                 if not self._deny_list_filter(new_zaak['decision'], ['buiten behandeling']):
                     continue
+                if new_zaak['title'].lower().startswith("*verwijder"):
+                    continue
 
-            elif f['text45'] in ['Vakantieverhuur', 'Vakantieverhuur afmelding', 'Vakantieverhuur vergunningsaanvraag']:
+            elif f['text45'] == 'Vakantieverhuur vergunningsaanvraag':
                 fields = [
                     {"name": "caseType", "from": "text45", "parser": to_string},
+                    {"name": "identifier", "from": 'mark', "parser": to_string},
                     {"name": "dateRequest", "from": "document_date", "parser": to_date},
+                    {"name": "processed", "from": "processed", "parser": to_string},
+                    {"name": "dateProcessed", "from": "date5", "parser": to_datetime},  # Datum afhandeling
+                    {"name": "location", "from": "text6", "parser": to_string},
+                    {"name": "status", "from": "title", "parser": to_string},
+                    {"name": "result", "from": "dfunction", "parser": to_string},
+                ]
+                new_zaak = _get_fields(fields, zaak)
+
+            elif f['text45'] == 'Vakantieverhuur':
+                fields = [
+                    {"name": "caseType", "from": "text45", "parser": to_string},
                     {"name": "identifier", "from": 'mark', "parser": to_string},
-                    {"name": "title", "from": 'subject1', "parser": to_string},
-                    {"name": "dateStart", "from": 'date6', "parser": to_date},  # Datum van
-                    {"name": "dateEnd", "from": 'date7', "parser": to_date},  # Datum tot
-                    {"name": "location", "from": 'text6', "parser": to_string},
-                    {"name": "status", "from": 'title', "parser": to_string},
+                    {"name": "dateRequest", "from": "document_date", "parser": to_date},
+                    {"name": "processed", "from": "processed", "parser": to_string},
+                    {"name": "dateProcessed", "from": "date5", "parser": to_datetime},  # Datum afhandeling
+                    {"name": "dateStart", "from": 'date6', "parser": to_date},  # Start verhuur
+                    {"name": "dateEnd", "from": 'date7', "parser": to_date},  # Einde verhuur
+                    {"name": "location", "from": "text6", "parser": to_string},
+                    # cancelled: true/false
+                    # cancelledDate: date(time)?
```

Comment:
Kun je deze`dateCancelled` noemen? Dat is in-lijn met de andere dateX properties